File: week04-1.py
##week04-1.py 二合一 py for迴圈版本
#leetcode 2579.count total number of colored cells
class Solution:
    def coloredCells(self, n: int) -> int:
        ans=0 #迴圈前面，ans先清空
        for i in range(1, n*2, 2):#py的for迴圈
            ans += i#把這些1，3，5，7的數，加入ans
        for i in range(1, n*2-1,2):
            ans += i#把這些1，3，5，的數，加入ans
        return ans

File: week04-1.c
// #week04-1.c  C 語言版 for迴圈版
// #LeetCode 2579. Count Total Numder of Colored Cells
long long coloredCells(int n) {
    long long ans = 0; // 很長很長的整數
    for(long long i=1; i<n*2; i+=2){
            ans += i;
        }
        for(long long i=1; i<n*2-1; i+=2){
            ans += i;
        }
        return ans;
}
